CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 1.2.0   | [PR#378](https://github.com/bbc/psammead/pull/378) Add psammead-storybook-helpers package |
```

Comment:
We dont need to bump/changelog for a change that doesnt reach outside of the packages dir :)

Reply:
```suggestion
```